content/troubleshooting/docs/reverse-proxy-sparkui.md

```diff
@@ -0,0 +1,290 @@
+# **Connect to SparkUI without Port Fowarding**
+
+This is an example of connecting to SparkUI running on Spark's driver pod via a reserve proxy solution, without an access to the kubectl tool or AWS console. We demostrate how to set it up via three EMR on EKS deployment methods: Spark Operator, JobRun API, and spark-submit.
+
+## Launch EMR on EKS jobs via Spark Operator
+
+There are 3 Steps to set up in EKS:
+
+1.Create a SparkUI reverse proxy and an ALB first. 
+
+The sample yaml file is in the [Appendix](#deploymentyaml). Make sure the EMR on EKS namespace at line #25 is updated by your namespace:
+```bash
+kubectl apply -f deployment.yaml
+```
+<div style="border: 1px solid red; padding: 10px; background-color: #f8d7da;">
+  <strong>NOTE:</strong> The example file is not production ready. The listen port 80 is not recommended. Make sure to stronger your Application Load Balance's security posture before deploy it to your production environment.
+</div>
+
+2.Submit two test jobs using EMR on EKS's Spark Operator. The sample job scripts [emr-eks-spark-example-01.yaml](#emr-eks-spark-example-01yaml) and [emr-eks-spark-example-02.yaml](#emr-eks-spark-example-02yaml) can be found in the Appendix section.
+
+Remember to specify the Spark configuration at line #16 `spark.ui.proxyBase: /sparkui/YOUR_SPARK_APP_NAME`, eg. `spark.ui.proxyBase: /sparkui/test-02`.
+```bash
+kubectl apply -f emr-eks-spark-example-01.yaml
+kubectl apply -f emr-eks-spark-example-02.yaml
+```
+
+3.Go to a web browser, then access their Spark Web UI while jobs are still running:
+
+```
+http://k8s-default-sparkui-2d325c0434-124141735.us-west-2.elb.amazonaws.com:80/sparkui/spark-example-01
```

Comment:
How to get this value? `k8s-default-sparkui-2d325c0434-124141735`

Reply:
it's the ingress endpoint from "kubectl get ingress" addressed in the following paragraph